maintenance_page/manifests/test/ingress_internal_to_main.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: apply-for-qts-test.test.teacherservices.cloud
+spec:
+  ingressClassName: nginx
+  rules:
+    - host: apply-for-qts-test.test.teacherservices.cloud
+      http:
+        paths:
+          - pathType: ImplementationSpecific
+            backend:
+              service:
+                name: apply-for-qts-test-web
+                port:
+                  number: 80
```

Comment:
I believe these should be apply-for-qts-test-web.test.teacherservices.cloud

Reply:
updated